packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -849,6 +850,17 @@ abstract class FlutterCommand extends Command<void> {
     );
   }
 
+  void usesFrontendServerStarterPathOption({required bool verboseHelp}) {
+    argParser.addOption(
+      FlutterOptions.kFrontendServerStarterPath,
+      help: 'When this value is provided, the frontend server will be started '
+            'in JIT mode from the specified file, instead of from the AOT '
```

Comment:
Does this need to be a dart source entrypoint, a JIT snapshot, or either?

Reply:
I was only considering the case of it being a Dart source entrypoint, but you're right that the AppJIT snapshot case is also supported. I've updated the help message.

Reply:
SGTM